App/Containers/TextilePhotos.js

```diff
@@ -1,22 +1,18 @@
 import React from 'react'
-import {View, Text, Image, TouchableWithoutFeedback, TouchableOpacity} from 'react-native'
-import ActionSheet from 'react-native-actionsheet'
+import {View, Text, Image } from 'react-native'
 import HeaderButtons, { Item } from 'react-navigation-header-buttons'
 import PhotoGrid from '../Components/PhotoGrid'
 import { connect } from 'react-redux'
 import Config from 'react-native-config'
 import PreferencesActions from '../Redux/PreferencesRedux'
 import TextileNodeActions, { ThreadData } from '../Redux/TextileNodeRedux'
-import * as TextileTypes from '../Models/TextileTypes'
+import * as TT from '../Models/TextileTypes'
```

Comment:
Would probably just be better to import the types you actually use, then the module alias isn't used at all.